### PR TITLE
Change the IPSLA feature to allow different metrics to be collected…

### PR DESCRIPTION
**Pull request by adaniels21487**
Change the IPSLA feature to allow different metrics to be collected per type.

- UDP Jitter metrics collected.
- Added opstatus code and DB field
- Discovery sets opstatus
- Added Status selector to SLA page
- Merged master and moved SQL

### CODE REVIEW

File: html/includes/graphs/device/sla.inc.php [hunk 1]
```diff
@@ -2,35 +2,19 @@
...
-$unit      = 'ms';
+$sla = dbFetchRow('SELECT `sla_nr` FROM `slas` WHERE `sla_id` = ?', array($vars['id']));
 
 $rrd_options .= ' -l 0 -E ';
-$rrd_options .= " COMMENT:'".str_pad($unit_long, $descr_len)."   Cur      Min     Max\\n'";
-
-$name = '';
-if ($sla['tag']) {
-    $name .= $sla['tag'];
+$rrd_filename = $config['rrd_dir']."/".$device['hostname']."/".safename('sla-'.$sla['sla_nr'].'.rrd');
```
Comment (murrant) [inline html/includes/graphs/device/sla.inc.php]: This should be 
```
rrd_name($device['hostname'], array('sla', $sla['sla_nr']));
```

**Comment by laf** [comment]
Needs the code updating to new format.

**Comment by adaniels21487** [comment]
Thanks @murrant @laf 
I'm on it.

File: html/pages/device/slas.inc.php [hunk 2]
```diff
@@ -79,4 +129,4 @@
     echo '</div>';
 }
 
-$pagetitle[] = 'SLAs';
+$pagetitle[] = 'SLAs';
```
Comment (murrant) [inline html/pages/device/slas.inc.php]: Oopsie, files should have a blank line at the end.

**Comment by murrant** [comment]
Looks, good.  I only read through, didn't get a chance to test.

**Comment by adaniels21487** [comment]
Thanks @murrant Fixed.

**Pull request merged**